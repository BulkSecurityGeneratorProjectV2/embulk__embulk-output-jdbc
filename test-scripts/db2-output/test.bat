CALL db2setcp
CALL db2 "CONNECT TO TESTDB USER db2admin USING XXXXXXXX"
CALL db2 "DROP TABLE EMBULK_OUTPUT"
CALL db2 "CREATE TABLE EMBULK_OUTPUT (ID DECIMAL(4) NOT NULL, NUM DECIMAL(12,2), STR CHAR(8), VARSTR VARCHAR(8), DT DATE, DTTM0 TIMESTAMP, DTTM3 TIMESTAMP(3), PRIMARY KEY(ID))"

CALL embulk run test.yml

CALL db2 "SELECT * FROM EMBULK_OUTPUT ORDER BY ID" > data\temp.txt

echo "diff data/test_expected.txt data/temp.txt"
diff data/test_expected.txt data/temp.txt

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-output-db2 FAILED!")
