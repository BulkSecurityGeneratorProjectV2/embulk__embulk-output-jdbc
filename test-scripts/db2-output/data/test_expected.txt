
ID     NUM            STR      VARSTR   DT         DTTM0                      DTTM3                  
------ -------------- -------- -------- ---------- -------------------------- -----------------------
    1.         123.40 test1    TEST1    2015-04-23 2015-04-23-16.02.03.000000 2015-04-23-16.02.03.123
    2.  1234567890.12 test9999 TEST9999 2015-12-30 2015-12-31-14.59.59.000000 2015-12-31-14.59.59.999
    3.              - -        -        -          -                          -                      

  3 レコードが選択されました。

